api-participant/src/main/resources/application.yml

```diff
@@ -8,8 +8,6 @@ env:
     name: ${DATABASE_NAME:pearl}
     password: ${DATABASE_USER_PASSWORD:dbpwd}
     user: ${DATABASE_USER:dbuser}
-  sam:
-    basePath: ${SAM_ADDRESS:https://sam.dsde-dev.broadinstitute.org}
   dsm:
```

Comment:
excellent, we'll list this on our `migration from DSP` accomplishments checklist for the month